<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>340</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>420</width>
    <height>340</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>420</width>
    <height>340</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AUST WiFi 自动重连工具</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background-color: #f5f5f5;
}

QWidget#centralwidget {
    background: transparent;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>15</number>
    </property>
    <property name="leftMargin">
     <number>20</number>
    </property>
    <property name="topMargin">
     <number>15</number>
    </property>
    <property name="rightMargin">
     <number>20</number>
    </property>
    <property name="bottomMargin">
     <number>15</number>
    </property>
    <item>
     <widget class="QLabel" name="titleLabel">
      <property name="styleSheet">
       <string notr="true">QLabel {
    font-size: 16px;
    font-weight: bold;
    color: #333;
    padding: 10px;
    background-color: white;
    border: 1px solid #ddd;
    border-radius: 6px;
}</string>
      </property>
      <property name="text">
       <string>AUST WiFi 管理器</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="statusLabel">
      <property name="styleSheet">
       <string notr="true">QLabel {
    font-size: 13px;
    color: #666;
    padding: 8px;
    background-color: white;
    border: 1px solid #ddd;
    border-radius: 4px;
}</string>
      </property>
      <property name="text">
       <string>状态: 正在检测网络连接...</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="infoLabel">
      <property name="styleSheet">
       <string notr="true">QLabel {
    font-size: 11px;
    color: #888;
    padding: 6px;
}</string>
      </property>
      <property name="text">
       <string>程序在后台运行，双击托盘图标显示/隐藏窗口</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="configButton">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>35</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: #0078d4;
    border: none;
    border-radius: 4px;
    color: white;
    font-size: 13px;
    font-weight: bold;
    padding: 8px;
}

QPushButton:hover {
    background-color: #106ebe;
}

QPushButton:pressed {
    background-color: #005a9e;
}</string>
      </property>
      <property name="text">
       <string>配置设置</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="controlLayout">
      <property name="spacing">
       <number>10</number>
      </property>
      <item>
       <widget class="QPushButton" name="startButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>32</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #107c10;
    border: none;
    border-radius: 4px;
    color: white;
    font-size: 12px;
    padding: 6px;
}

QPushButton:hover {
    background-color: #0e6e0e;
}

QPushButton:pressed {
    background-color: #0c5d0c;
}</string>
        </property>
        <property name="text">
         <string>开始</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stopButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>32</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #d13438;
    border: none;
    border-radius: 4px;
    color: white;
    font-size: 12px;
    padding: 6px;
}

QPushButton:hover {
    background-color: #b52d31;
}

QPushButton:pressed {
    background-color: #9a272a;
}</string>
        </property>
        <property name="text">
         <string>停止</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="styleSheet">
    <string notr="true">QStatusBar {
    background-color: #f0f0f0;
    border-top: 1px solid #ddd;
    color: #666;
    font-size: 10px;
}</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
